src/core/request.ts

```diff
@@ -23,6 +23,10 @@ import JSONbigInt = require('json-bigint');
 
 const JSONbigString = JSONbigInt({ storeAsString: true });
 
+import debug from 'debug';
+
+const _requestDebug = debug('ig:request');
```

Comment:
Underscore prefix in var names is old-fashioned JS style to mark variable as private. We're in TS and we do have `private` modifier now. Let's declare debug as
```ts
private debug = debug('ig:request');
```

In every class that uses it.